ui/app/AppLayouts/Profile/views/wallet/NetworksView.qml

```diff
@@ -35,97 +97,84 @@ Item {
         width: parent.width
         spacing: 0
 
-        Repeater {
-            id: layer1List
-            model: SortFilterProxyModel {
-                sourceModel: root.combinedNetworks
-                filters: ValueFilter {
-                    roleName: "layer"
-                    value: 1
-                }
+        StatusTabBar {
+            id: testModeViewTabBar
+            objectName: "testModeViewTabBar"
+            StatusTabButton {
+                text: qsTr("Mainnet")
+                objectName: "testModeViewMainButton"
+                width: implicitWidth
             }
-            delegate: WalletNetworkDelegate {
-                readonly property var network: {
-                    let chainId = root.areTestNetworksEnabled ? model.testChainId : model.prodChainId
-                    return ModelUtils.getByKey(root.flatNetworks, "chainId", chainId)
-                }
-                
-                objectName: "walletNetworkDelegate_" + network.chainName + '_' + network.chainId
-                areTestNetworksEnabled: root.areTestNetworksEnabled
-                chainName: network.chainName
-                iconUrl: network.iconUrl
-                
-                onClicked: editNetwork(model)
+            StatusTabButton {
+                text: qsTr("Testnet")
+                objectName: "testModeViewTestButton"
+                width: implicitWidth
             }
         }
 
-        Separator {
-            height: Theme.padding
-        }
-
-        StatusSectionHeadline {
-            leftPadding: Theme.padding
-            rightPadding: Theme.padding
-            text: qsTr("Layer 2")
-            topPadding: Theme.smallPadding
-            bottomPadding: Theme.smallPadding
-        }
-
         Repeater {
-            id: layer2List
+            id: networkList
             model: SortFilterProxyModel {
-                sourceModel: root.combinedNetworks
+                sourceModel: root.flatNetworks
                 filters: ValueFilter {
-                    roleName: "layer"
-                    value: 2
+                    roleName: "isTest"
+                    value: testModeViewTabBar.currentIndex == root.testnetTabIndex
                 }
+                sorters: [
+                    RoleSorter {
+                        roleName: "isActive"
```

Comment:
@dlipicar do we really want to move network items up/down in the list based on this property (selection)? Not sure if that's a good UX.